Mathlib/CategoryTheory/Limits/Shapes/BinaryProducts.lean

```diff
@@ -728,13 +728,11 @@ end ProdLemmas
 
 noncomputable section CoprodLemmas
 
--- @[reassoc (attr := simp)]
-@[simp] -- Porting note: removing reassoc tag since result is not hygienic (two h's)
```

Comment:
A failure of `@[reassoc]` sounds like a note worth keeping.

Reply:
Actually, it's working fine now, so I'll remove the hand-written lemma instead.